crowbar_framework/vendor/assets/javascripts/jquery/ledUpdate.js

```diff
@@ -268,6 +268,12 @@
 
   LedUpdate.prototype.destroy = function() {
     $('.led.unready, .led.in_process, .led.spin').destroy();
+    // workaround https://bugzilla.suse.com/show_bug.cgi?id=949273
+    // to stop the green led from moving after node is in ready state
+    $('.led.ready').sprite({
```

Comment:
Should we also do that for the others (.led.failed, .led.pending, .led.waiting, led.red)?